securedrop/tests/test_db.py

```diff
@@ -4,24 +4,10 @@
 from mock import MagicMock
 
 from .utils import db_helper
-from models import (Journalist, Submission, Reply, Source, get_one_or_else,
+from models import (Journalist, Submission, Reply, get_one_or_else,
                     LoginThrottledException)
 
 
-def test_source_public_key_setter_unimplemented(journalist_app, test_source):
```

Comment:
If we're removing the explicit setter and deleter for `Source.public_key`, we should still test that it can't be set or deleted.

Reply:
I can add it back if needed, but this seems overly specific as a test case? Would you add this same test for every field declared as a `@property` in the securedrop code base?